examples/poyo_plus/scripts/generate_config.py

```
@@ -0,0 +1,388 @@
+    "595806300",
+    "689388034",
+    "649938038",
+    "645689073",
+    "510514474",
+    "505695962",
+    "512326618",
+    "562122508",
+    "653122667",
+}
+
+def read_task_contents(csv_path: str, holdout_sessions: set = None) -> Dict[str, Dict[str, bool]]:
```

Ruff (0.15.10)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)